frontend/src/metabase/parameters/components/ValuesSourceModal/ValuesSourceCardModal.tsx

```diff
@@ -72,7 +73,7 @@ type ModalProps = ModalOwnProps &
   ModalDispatchProps;
 
 const ValuesSourceCardModal = ({
-  name,
+  parameter,
```

Comment:
Wonder why we need to change this parameter since we still don't use any other key than `parameter.name`.